jobs/FunctionTest/prepare_manifest.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash -ex
-REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog")
-
+REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog" "image-service")
```

Comment:
I don't know why Alan's comment is gone:
In the previous edition of the code review @changev suggested adding an if else here so that the image-service manifest isn't loaded each time. I found it a good idea, yet after took a deeper consideration, I found that if put an if else here, then its better to classify whether image service should be included in  manifest-pr-gate.json and rackhd-pm2-config.yml as well. As it won't do any obvious benefit , I plan to leave it as it is for now. And in the near future, if time permits,  I can think of better way to handle this.